hexrd/findorientations.py

```diff
@@ -552,6 +591,47 @@ def generate_eta_ome_maps(cfg, hkls=None, save=True):
     return eta_ome
 
 
+def _filter_eta_ome_maps(eta_ome, filter_stdev=False):
+    """
+    Apply median and gauss-laplace filtering to remove streak artifacts.
+
+    Parameters
+    ----------
+    eta_ome : TYPE
+        DESCRIPTION.
+    filter_stdev : TYPE, optional
+        DESCRIPTION. The default is 1.
+
+    Returns
+    -------
+    eta_ome : TYPE
+        DESCRIPTION.
+
+    """
+    gl_filter = ndimage.filters.gaussian_laplace
+    for i, pf in enumerate(eta_ome.dataStore):
+        # first compoute row-wise median over omega channel
+        ome_median = np.tile(np.median(pf, axis=0), (len(pf), 1))
+
+        # subtract
+        # !!! this changes the reference obj, but fitlering does not
+        pf -= ome_median
+
+        # filter
+        # !!! False/None: only row median subtraction
+        #     True: use default
+        #     scalar: stdev for filter in pixels
+        # ??? simplify this behavior
+        if filter_stdev:
+            if isinstance(filter_stdev, bool):
+                filter_stdev = filter_stdev_DFLT
+            pf = -gl_filter(pf, filter_stdev)
```

Comment:
It doesn't matter very much, but you might be able to modify `pf` in place via
`pf[:] = -gl_filter(pf, filter_stdev)`

Reply:
We can probably use the `output` kwarg in `gauss_laplace` 

https://docs.scipy.org/doc/scipy-0.14.0/reference/generated/scipy.ndimage.filters.gaussian_laplace.html#scipy.ndimage.filters.gaussian_laplace

Reply:
have you tried the `[:]` syntax?

Reply:
> have you tried the `[:]` syntax?

Yeah, it works.